### ISSUE TITLE
Missing XMLWriter

### ISSUE THREAD

**Issue by Rvice** [thomas-rcv/admidio-written_communications#23]
I'm getting a 500 error when trying to use the standard template after install. I can access the menus and select who I want to generate the document for, but on pressing Download Document I get the 500 error.

I checked my webserver logs and I'm seeing the following:

```
Stack Tracer:
0. ./admidio/adm_plugins/written_communications/classes/PHPWord/Writer/Word2007/ContentTypes.php(37): PHPWord_Shared_XMLWriter->__construct(1)

1. /admidio/adm_
20211213T170338: ./adm_plugins/written_communications/written_communications_functions.php
PHP Fatal error:  Uncaught Error: Class 'XMLWriter' not found in ./admidio/adm_plugins/written_communications/classes/PHPWord/Shared/XMLWriter.php:61
```

[Update 2022-01-11] Updated to Written Communications 3.4.0 with Admidio 4.1.2 with the same results

**Comment by Rvice** [comment]
FYI, updated to 3.4.2 and have a new error with a "fresher" install. I deleted the plugin folder and replaced it with the 3.4.2 version. Then just went and used the template `demo_Email_en` with the following options. Linked to [Issue 27](https://github.com/thomas-rcv/admidio-written_communications/issues/27) for the new problem.

**Comment by Fasse** [comment]
So with the new version this error is obsolet?

**Comment by Fasse** [comment]
Closed because this was solved in 3.4.2
